README.md

```diff
@@ -1783,10 +1783,10 @@ RMB supports optimistic locking. By default it is disabled. Module developer can
 
 * `off` Optimistic Locking is disabled. The trigger is removed if it existed before.
 * `failOnConflict` Optimistic Locking is enabled. Version conflict will fail the transaction
-* `logOnConflict` Optimistic Locking is enabled. Version conflict info is logged as a warning message with a customized SQL error code 23F09. Consult [Vertx logging](https://vertx.io/docs/vertx-core/java/#_logging) and [PostgreSQL Errors and Messages](https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html) if `logOnConflict` doesn't log. A sample log entry looks like below.
+* `logOnConflict` Optimistic Locking is enabled. Version conflict info is logged with a customized SQL error code 23F09. Consult [Vertx logging](https://vertx.io/docs/vertx-core/java/#_logging) and [PostgreSQL Errors and Messages](https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html) if `logOnConflict` doesn't log. A sample log entry looks like below.
 
 ```
-14:26:54 [] [] [] [] WARN  ?                    Backend notice: severity='WARNING', code='23F09', message='Cannot update record 57db089f-18e4-7815-55d5-4cc6607e9059 because it has been changed: Stored _version is 2, _version of request is "1"' ...
+14:26:54 [] [] [] [] WARN  ?                    Backend notice: severity='NOTICE', code='23F09', message='Cannot update record 57db089f-18e4-7815-55d5-4cc6607e9059 because it has been changed: Stored _version is 2, _version of request is "1"' ...
```

Comment:
Can we add a fail log example? Can we avoid "Cannot update" when we actually update (logOnConflict)? "optimistic locking" should be mentioned to help people find our optimistic locking documentation.

_Sample log entries:_
```
14:26:54 [] [] [] [] ERROR  ?                    Backend notice: severity='EXCEPTION', code='23F09', message='Cannot update record 57db089f-18e4-7815-55d5-4cc6607e9059 because it has been changed (optimistic locking): Stored _version is 2, _version of request is "1"' ...
```
```
14:26:54 [] [] [] [] WARN  ?                    Backend notice: severity='NOTICE', code='23F09', message='Ignoring optimistic locking conflict while overwriting changed record 57db089f-18e4-7815-55d5-4cc6607e9059: Stored _version is 2, _version of request is "1"' ...
```

After changing the code please copy and paste actual log entries.

Reply:
The log entry `failOnConflict` becomes an ERROR entry from PostgreSQL like this:
```
01:32:20 [] [] [] [] ERROR PostgresClient       saveBatch size=2 { "message": "version conflict", "severity": "ERROR", "code": "23F09", "where": "PL/pgSQL function raise_409() line 1 at RAISE", "file": "pl_exec.c", "line": "3337", "routine": "exec_stmt_raise" }
```